server/api.js

```diff
@@ -25,7 +29,7 @@ const client = new AuthorizationCode({
 
 const authorizationUri = client.authorizeURL({
 	//we can put in the redirect_uri when we deploy the app
-	redirect_uri: 'http://localhost:3100/api/callback',
+	redirect_uri: 'https://oauth-frontend-cyf.herokuapp.com/createprofile',
```

Comment:
you don't you use the /callback URI in the api.js, do you? In this case the code should be removed here, because it can be confusing (at least it was for me :-)  ) what happens and where